src/editor-sidebar/screen-header.js

```diff
@@ -17,7 +17,7 @@ import { chevronRight, chevronLeft } from '@wordpress/icons';
 
 const ScreenHeader = ( { title, onBack } ) => {
 	// TODO: Remove the fallback component when the minimum supported WordPress
-	// version was increased to 6.7.
+	// version was increased to 6.8.
 	const BackButton = Navigator?.BackButton || NavigatorToParentButton;
```

Comment:
It turns out that `Navigator.BackButton` is not yet exposed in WordPress 6.7. Therefore, we found that the fallback component is necessary as long as CBT supports WordPress 6.7.

In the future, when we bump the minimum required WordPress version to 6.8, we will be able to remove this fallback code.